overlord/snapstate/snapstate.go

```diff
@@ -1349,6 +1349,13 @@ func UpdateMany(ctx context.Context, st *state.State, names []string, userID int
 	return updateManyFiltered(ctx, st, names, userID, nil, flags, "")
 }
 
+// EnforceSnaps installs/updates/removes snaps reported by validErr. validationSets is the list of sets passed by the user and it's used in the final
+// stage to update validation-sets tracking in the state.
```

Comment:
Please also specify what userID is used for.

Reply:
Added a short note. FWTW we don't seem to be documenting it in any of the snapstate functions, it's one of the "standard" arguments similar to state.State.